pkg/yamltemplate/filetests/ytt-library/assert-equals-err-decimal.yml

```diff
@@ -0,0 +1,11 @@
+#@ load("@ytt:assert", "assert")
+#@ load("@ytt:json", "json")
+
+test_equals: #@ assert.equals(3.14159, 2.71828)
```

Comment:
Love these additional tests! Now that there are several files what do you think about placing all these assert tests into an assert directory?